requirements-dev.txt

```diff
@@ -2,5 +2,7 @@
 addict
```

Comment:
curious why do dev need the addict?

Reply:
Not sure why this was added. I can remove this.